.gitignore

```diff
@@ -26,6 +26,7 @@ var/
 *.egg
 *.mypy_cache/
 logs/
+output/
```

Comment:
If you keep the default `edr_output`, is it supposed to be `edr_output`?

Reply:
Good catch missed it, done

Reply:
Since we dont have edr_output default anymore and its going to be os.getcwd(), so removed that